### PR TITLE
[add] support for relative host directories when using bind mounts

### PR DESCRIPTION
**Pull request by sescobb27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,10 @@ Then if you run `ls ~/test` you should see a file named `myfile.txt` with
 
 - Doesn't support relative paths in the container when `COPY`ing
   - `COPY ./relative/path/to/origin:/absolute/path/to/destination`
-- Doesn't support building `VOLUMES` only [Bind Mounts](https://docs.docker.com/storage/bind-mounts/)
+- Doesn't support standard `VOLUMES`, it only supports the following `VOLUME`s type
+with custom syntax
+  - [Bind Mounts](https://docs.docker.com/storage/bind-mounts/) e.g `VOLUME ~/path/to/host/dir:/path/to/container/dir`
+  - [Named Modules](https://docs.docker.com/storage/volumes/) e.g `VOLUME volume_name` and then using it like `VOLUME volume_name:/path/to/container/dir`
```
Comment (ghost) [inline README.md]: Isn't this mean to be "Named Volumes" ?

**Comment by ghost** [comment]
We should get travis set up for this

**Comment by sescobb27** [comment]
what do you mean? travis is already set up and passing

**Comment by ghost** [comment]
Sorry - page hadn't rendered properly

File: README.md (hunk 1 above)
Comment (sescobb27) [reply README.md]: hahaahaha ups, typo

**Comment by ghost** [comment]
Actually I'm seeing some different behavior locally (OSX and on Centos 7) - mix test fails with the following : 

```
HEAD is now at 70656b5 [add] support for relative host directories when using bind mounts                                    [10/4757]
[vagrant@localhost ex_docker_build]$ mix test
........                           

  1) test tar archives tar a directory (ExDockerBuild.TarTest)
     test/tar_test.exs:18
     Assertion with == failed
     code:  assert :erl_tar.table(path) == {:ok, ['Dockerfile_bind.dockerfile', 'Dockerfile_erlang.dockerfile', 'Dockerfile_simple.doc
kerfile']}
     left:  {:ok, ['Dockerfile_bind.dockerfile', 'Dockerfile_simple.dockerfile', 'Dockerfile_erlang.dockerfile']}
     right: {:ok, ['Dockerfile_bind.dockerfile', 'Dockerfile_erlang.dockerfile', 'Dockerfile_simple.dockerfile']}
     stacktrace:                    
```

**Comment by ghost** [comment]
If I configure so remove is not executed - like so : 


```
defmodule ExDockerBuild.TarTest do
  use ExUnit.Case, async: true

  alias ExDockerBuild.Tar

  @cwd System.cwd!()
  @fixtures_path Path.join([@cwd, "test", "fixtures"])

  setup do
    on_exit(fn ->
 #     Path.wildcard("*.tar") |> Enum.map(&File.rm!/1)
    end)
```
This is what's in the tar - perhaps there's a different, or occasionally random sorting applied 

```
[vagrant@localhost ex_docker_build]$ tar tvfp ./fixtures.tar 
-rw-rw-r-- 1000/1000       127 2018-10-04 17:49 Dockerfile_bind.dockerfile
-rw-rw-r-- 1000/1000       530 2018-10-04 17:49 Dockerfile_simple.dockerfile
-rw-rw-r-- 1000/1000      2548 2018-10-04 17:49 Dockerfile_erlang.dockerfile
```

Maybe apply a sort to list on both side of comparison before comparing?

**Comment by sescobb27** [comment]
I just push the fix for it, can you try it out?

**Comment by ghost** [comment]
Works for me . Merging .

**Pull request merged**